docs/documentation/install/install-the-kit.md

```diff
@@ -39,16 +39,24 @@ Learning a few basic terminal commands can make using the kit much easier.
 
 You need to navigate to your prototype in the terminal. Most commands for the kit need to be run in the prototype folder.
 
-#### Mac users:
+If you're using a Mac, enter:
+
 ```
 cd ~/projects/juggling-licence-prototype
 ```
 
-#### Windows users:
+If you're using Windows, enter:
+
 ```
 cd ~/Documents/projects/juggling-licence-prototype
 ```
 
+You must add quotation marks around everything after `~/` if any of your folder names contain spaces. For example:
```

Comment:
is it clearer to flip the context to the start?

> If any of your folder names contain spaces, you'll need to add quotation marks around everything after `~/`. For example:

when I'm coding I avoid using spaces in my folders to keep it simple, is it worth adding that?

Reply:
Good idea - will tweak!

On your second point - adding that would be slightly outside the user need of this change, which is that the user is following the specific instructions on this page but can't `cd`,  because they're inside a shared department folder they don't have name changing rights to.